spacewalk/setup/lib/Spacewalk/Setup.pm

```diff
@@ -42,6 +42,9 @@ use constant POSTGRESQL_SCHEMA_FILE => File::Spec->catfile("/etc", "sysconfig",
 use constant POSTGRESQL_DEPLOY_FILE => File::Spec->catfile("/etc", "sysconfig",
     'rhn', 'postgres', 'deploy.sql');
 
+use constant WEBSOCKIFY_CONF_FILE => File::Spec->catfile("/etc", "sysconfig",
```

Comment:
Nit: is there a reason for mixing `"` and `'`? :)

Reply:
I don't think so... just habit: I'll change

Reply:
in fact this was mainly due to a copy paste of code that already had a mix ;)

Reply:
Change dropped since the config file is no longer needed